sim/src/analytics.rs

```diff
@@ -43,6 +43,16 @@ pub struct Analytics {
     // TODO Hack: No TripMode means cancelled
     // Finish time, ID, mode, trip duration
     pub finished_trips: Vec<(Time, TripID, Option<TripMode>, Duration)>,
+
+    // Records how long was spent waiting at each turn (Intersection) for a given trip
+    // Over a certain threshold
+    // TripID, [(TurnID, Time Waiting In Seconds)]
+    pub trip_intersection_delays: BTreeMap<TripID, Vec<(TurnID, u8)>>,
+    // Records how long was spent waiting at each lane for a given trip
+    // Over a certain threshold
+    // TripID, [(LaneID, Time Waiting In Seconds)]
```

Comment:
Comment is misleading, this is a percentage in [0, 100]

Reply:
Yup, again good spot. Thanks